lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb

```diff
@@ -184,7 +184,7 @@ def self.get_library_version_from_gradle_config(import_key:)
 
           File.open(gradle_file_path, 'r') do | f |
             text = f.read
-            text.match(/#{Regexp.escape(import_key)}\s?=\s?'(.*?)'/m)&.captures&.first
+            text.match(/#{Regexp.escape(import_key)}\s?=\s?['"](.*?)["']/m)&.captures&.first
```

Comment:
👍

Reply:
>what if the build.gradle file contains more than one whitespace between the key and the = sign, or between = and the value? These should be \s* imho.

It shouldn't have more than one whitespace tbh, but there's no harm with changing the regex. 

> what if there's a line containing the import key but not starting with it? I think the regex should start with ^\s* to allow leading whitespaces but no other prefix or chars before.

Hmm... not sure about this now. There may be cases where the import key we want to pass has not the prefix part. Let's keep it as it is now and iterate if required when the whole setup is completed on the Gutenberg repository. We'll have a better picture at that point.

Reply:
Ah I see, didn't realize this was intentional that you might want to pick a key… that might or might not have a prefix. Makes sense then 👍